src/gardenlinux/github/__main__.py

```diff
@@ -0,0 +1,836 @@
+
+    s3_artifacts._bucket.download_file(
+        release_object.key, artifacts_dir.joinpath(f"{cname}.s3_metadata.yaml")
+    )
+
+
+def download_all_metadata_files(version, commitish):
+    repo = Repo(".")
+    commit = repo.commit(commitish)
+    flavors_data = commit.tree["flavors.yaml"].data_stream.read().decode("utf-8")
+    flavors = FlavorsParser(flavors_data).filter(only_publish=True)
+
+    local_dest_path = Path("s3_downloads")
+    if local_dest_path.exists():
+        shutil.rmtree(local_dest_path)
+    local_dest_path.mkdir(mode=0o755, exist_ok=False)
+
+    s3_artifacts = S3Artifacts(GARDENLINUX_GITHUB_RELEASE_BUCKET_NAME)
+
+    commitish_short = commitish[:8]
+
+    for flavor in flavors:
+        cname = CName(flavor[1], flavor[0], "{0}-{1}".format(version, commitish_short))
+        LOGGER.debug(f"{flavor=} {version=} {commitish=}")
+        # Filter by image variants - only download if the flavor matches one of the variants
+        flavor_matches_variant = False
+        for variant_suffix in IMAGE_VARIANTS:
+            if variant_suffix == "":
+                last_part = cname.cname.split("-")[-1]
+                if "_" not in last_part:
+                    flavor_matches_variant = True
+                    break
+            elif variant_suffix in cname.cname:
+                # Specific variant (any non-empty string in IMAGE_VARIANTS)
+                flavor_matches_variant = True
+                break
+
+        if not flavor_matches_variant:
+            LOGGER.info(
+                f"Skipping flavor {cname.cname} - not matching image variants filter"
+            )
+            continue
+
+        try:
+            download_metadata_file(
+                s3_artifacts, cname.cname, version, commitish_short, local_dest_path
+            )
+        except IndexError:
+            LOGGER.warn(f"No artifacts found for flavor {cname.cname}, skipping...")
+            continue
+
+    return [str(artifact) for artifact in local_dest_path.iterdir()]
+
+
+def _parse_match_section(pkg_list: list):
+    output = ""
+    for pkg in pkg_list:
+        # If is dict, the package has additional information relevant for release notes
+        if isinstance(pkg, dict):
+            pkg_string = next(iter(pkg))
+            output += f"\n{pkg_string}:\n"
+            for item in pkg[pkg_string]:
+                for k, v in item.items():
+                    output += f"  * {k}: {v}\n"
+    return output
+
+
+def release_notes_changes_section(gardenlinux_version):
+    """
+    Get list of fixed CVEs, grouped by upgraded package.
+    Note: This result is not perfect, feel free to edit the generated release notes and
+    file issues in glvd for improvement suggestions https://github.com/gardenlinux/glvd/issues
+    """
+    try:
+        url = f"https://glvd.ingress.glvd.gardnlinux.shoot.canary.k8s-hana.ondemand.com/v1/patchReleaseNotes/{gardenlinux_version}"
+        response = requests.get(url, timeout=REQUESTS_TIMEOUTS)
```

Comment:
As mentioned above please replace the `REQUESTS_TIMEOUTS` here by primitive ones. E.g. `( REQUESTS_DEFAULT_CONNECT_TIMEOUT, REQUESTS_DEFAULT_READ_TIMEOUT )`. Are the default timeout values of the library not suitable?